Source/Core/DolphinQt2/Host.cpp

```diff
@@ -109,6 +110,10 @@ void Host_YieldToUI()
 
 void Host_UpdateDisasmDialog()
 {
+  RunOnObject(QApplication::instance(), [&] {
+    emit Host::GetInstance()->UpdateDisasmDialog();
+    return true;
```

Comment:
Why do you return true here?

Reply:
Cause I once tried to not return anything and it was weird, but I think it's okay now, not sure what happened there

Reply:
Actually, nvm, it doesn't let me return nothing, it really wants me to return soemthing so I jsut put true there.